.github/workflows/update_v8_snapshot_cache.yml

```diff
@@ -1,6 +1,8 @@
 name: Update V8 Snapshot Cache
 on:
   schedule:
+    # Run everyday at 00:00 UTC
+    - cron: '0 0 * * *'
     # Run every Wednesday at 00:00 UTC
     - cron: '0 0 * * 3'
```

Comment:
Still need this chron if you already run every day at 00:00 UTC?

Reply:
Doh. Good call. I updated this to be correct with what I'm thinking. The logic is now:

* Every day except wednesday: iterative regeneration
* Wednesday: full regeneration
* Release branches: every commit